code-of-conduct.md

```diff
@@ -0,0 +1,3 @@
+**Kubernetes Community Code of Conduct**
+
+Kubernetes follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
```

Comment:
This should say "Kubernetes Code of Conduct". Ref: https://github.com/kubernetes/kubernetes-template-project/blob/master/code-of-conduct.md